src/c/core/session/session_info.c

```diff
@@ -53,6 +54,12 @@ void uxr_buffer_create_session(
     payload.client_representation.client_key.data[3] = info->key[3];
     payload.client_representation.session_id = info->id;
     payload.client_representation.optional_properties = false;
+#ifdef UCLIENT_PROFILE_INTERPROCESS
+    payload.client_representation.optional_properties = true;
+    payload.client_representation.properties.size = 1;
+    payload.client_representation.properties.data[0].name = "uxr_ip";
```

Comment:
https://github.com/eProsima/Micro-XRCE-DDS-Agent/pull/234

Reply:
Is this solved @Acuadros95 ?

Reply:
The property and flag name are already updated on this pull request:
https://github.com/eProsima/Micro-XRCE-DDS-Client/blob/6899dc36f7795243f76d5181fbebf39a1169bd69/src/c/core/session/session_info.c#L57-L62

Reply:
Nice, thanks